soh/soh/Enhancements/randomizer/3drando/item_pool.cpp

```diff
@@ -624,6 +624,30 @@ static void PlaceVanillaOverworldFish() {
   }
 }
 
+static void PlaceFreestandingItems() {
+  auto ctx = Rando::Context::GetInstance();
+  auto option = ctx->GetOption(RSK_SHUFFLE_FREESTANDING);
+  for (RandomizerCheck loc : ctx->GetLocations(ctx->allLocations, RCTYPE_FREESTANDING)) {
+    RandomizerGet vanillaItem = Rando::StaticData::GetLocation(loc)->GetVanillaItem();
+    if (option.Is(RO_TOKENSANITY_OVERWORLD) || option.Is(RO_TOKENSANITY_ALL)) {
```

Comment:
I don't think a tokensanity check is supposed to here.

Reply:
looks like freestanding never got it's own option enum and just copied tokens, fixing.

Reply:
I figured that "dungeon / overworld" choices would be pretty common and so it would make more sense to have a single enum for them, ideally with a name not so directly tied to tokensanity. I'm pretty sure potsanity also uses the same distinction, but I think it just copies the enum rather than using the existing instance.

Reply:
With the recent commit, there are now 3 functionally and mostly-semantically identical enums. Why have 3 instead of just 1?

Reply:
It's about intention and code clarity. Enums don't have any additional impact on performance as they just evaluate to the same after compiling. Every option has its own enums even though a large portion will have overlap.